tests/utils_tests/test_lorem_ipsum.py

```diff
@@ -1,16 +1,106 @@
 import unittest
+from unittest.mock import patch
 
-from django.utils.lorem_ipsum import paragraphs, words
+from django.utils.lorem_ipsum import paragraph, paragraphs, sentence, words
 
 
 class WebdesignTest(unittest.TestCase):
 
-    def test_words(self):
+    def test_negative_words(self):
+        """Common words when passing negative words"""
+        self.assertEqual(words(-5), 'lorem ipsum dolor sit amet consectetur adipisicing elit sed do eiusmod '
```

Comment:
Use hanging indent as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). For example, this would look like this:
```
self.assertEqual(
    words(-5),
    'lorem ipsum dolor sit amet consectetur adipisicing elit sed do '
    'eiusmod tempor incididunt ut'
)
```
There are other places in this PR to fix for this style. I won't comment on each one.